Client/Frontend/Intro/IntroViewController.swift

```diff
@@ -189,6 +194,13 @@ class IntroViewController: UIViewController, UIScrollViewDelegate {
             make.bottom.equalTo(syncCardView)
         }
 
+        // We need a refrence to the sync pages textlabel so we can adjust the constraints
```

Comment:
s/refrence/reference/

Reply:
I need a spell checker for my comments 😢